### PR TITLE
Add avatar to the main page

### PR DESCRIPTION
**Pull request by aburgel**
domain @jonmauney

task https://app.asana.com/0/346606283875063/347518883925815
screen shot 2017-05-23 at 7 13 41 pm

Added the image for the logged in user to the index page.

### CODE REVIEW

File: app/views/hello/index.html.erb [hunk 1]
```diff
@@ -1 +1,4 @@
 <h1>Hello, Winos!</h1>
+<div>
+  <%= image_tag current_user.picture_url %>
```
Comment (jonmauney) [inline app/views/hello/index.html.erb:3]: this is good, but I believe the best practice is:
```html
<img src="http://bit.ly/2qTCxUs" />
```

**Comment by jonmauney** [comment]
one very important comment, we don't want to get our webapp off on the wrong foot.

<<domainTAFN

**Comment by while1malloc0** [comment]
domain lgtm

**Comment by while1malloc0** [comment]
💯

**Pull request merged**